hera_cal/utils.py

```diff
@@ -144,3 +145,60 @@ def get_aa_from_calfile(freqs, calfile, **kwargs):
 
     # generate aa
     return get_aa(freqs, **kwargs)
+
+
+def JD2LST(JD, longitude):
```

Comment:
Make longitude a keyword argument with default value that of the center of the array.